_includes/v21.1/orchestration/start-cockroachdb-helm-insecure.md

```diff
@@ -94,5 +112,5 @@ The CockroachDB Helm chart is undergoing maintenance for compatibility with Kube
     ~~~
 
 {{site.data.alerts.callout_success}}
-The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to a pod/node's logs to troubleshoot, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
+The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to logs for a pod or node, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
```

Comment:
"a pod or node" now makes me realize the original wording (not mine, AFAIK) may have meant "CockroachDB pod, aka CockroachDB node" and not two separate entities. Unless you can also access the K8s node (VM) logs this way, maybe this should just say "a pod"?